docs/source/releasing.rst

```diff
@@ -22,6 +22,16 @@ If you are bumping to a .dev0 suffix, omit :code:`--tag` and only pass :code:`--
 
     $ ./scripts/release bump v0.4.0.dev0 --commit
 
+By default, the release script will refuse to downgrade the version. This can be
+overridden with the :code:`--force` option, for example to prepare a point
+release. In that case the repository should be prepared to contain only the
```

Comment:
In other words, it should only be done on the master branch if no unrelated changes have been merged yet, otherwise a stable branch should be used

Reply:
Ok, will add!